Core/Newsletter2Go/Components/Api/Resource/NewsletterCustomer.php

```diff
@@ -325,6 +325,7 @@ public function getCustomerFields()
      * @return array
      */
     public function getStreamList(
+        $subscribed = false,
```

Comment:
`$subscribed` is  an optional parameter and `$group` is not. Mandatory parameters should be placed first in method signature, like:
```PHP
public function getStreamList($group, $subscribed = false, $emails = [] ...)
```
or we keep the order like this and remove `$subscribed` default value.

Reply:
changed order and add subscribed at the end